website/content/en/docs/kubebuilder/operator-scope.md

```diff
@@ -0,0 +1,285 @@
+    MetricsBindAddress: metricsAddr,
+    Port:               9443,
+    LeaderElection:     enableLeaderElection, 
+    LeaderElectionID:   "f1c5ece8.example.com",
+})
+...
+```
+
+**NOTE** The above example is the default code generated with `operator-sdk init` which initializes the Manager to be cluster-scoped.
+
+## Watching resources in a single Namespace
+
+To restrict the scope of the [Manager's][ctrl-manager] cache to a specific Namespace set the Namespace attribute from [Options][ctrl-options]:
+
+```go
+...
+mgr, err = ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
+    Scheme:             scheme,
+    Namespace:          "operator-namespace",
+    MetricsBindAddress: metricsAddr,
+})
+...
+``` 
+
+## Watching resources in a set of Namespaces
+
+It is possible to use the [MultiNamespacedCacheBuilder][multi-namespaced-cache-builder] attribute from 
+[Options][ctrl-options] to watch and manage resources only in a specific set of Namespaces:
+
+```go
+...
+namespaces := []string{"foo", "bar"} // List of Namespaces
+
+mgr, err = ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
+    Scheme:             scheme,
+    NewCache:           cache.MultiNamespacedCacheBuilder(namespaces),
+    MetricsBindAddress: fmt.Sprintf("%s:%d", metricsHost, metricsPort),
+})
+...
+```
+
+In the above example, if a CR(CustomResource), which is managed by its controllers, is applied to a Namespace that is not in this List of Namespaces then, its controller will not re-trigger its reconcile since the Namespaced is NOT part of the [Manager][ctrl-manager] scope.
+
+**IMPORTANT:** Note that this is not intended to be used for excluding Namespaces, this is better done via a Predicate. Also note that you may face performance issues when using this with a high number of Namespaces.
+
+## Restricting Roles and permissions
+
+The operator-scope defines the [Manager][ctrl-manager] cache scope but not the permissions to access the resources. After updating the Manager's scope to be Namespaced, the [Kubernetes Role-Based Access Control or the (RBAC)][k8s-rbac] permissions should be restricted accordingly.
+
+These permissions are found in the directory `config/rbac/`. The `ClusterRole` and `ClusterRoleBinding` manifests are responsible for setup the permissions which allow the operator to have access to the resources.
+
+**NOTE** The other files that can be found in this directory not requires changes to achieve this goal. The RBAC manifests with the names `<kind>_editor_role.yaml` and `<kind>_viewer_role.yaml` are not used by the operator and are meant to be helpers for specific config to allow access to users and clients per Kind. Also, the files with the names `auth_proxy_*.yaml` are specific to the [kube-auth-proxy][kube-rbac-proxy]. 
+
+### Changing the permissions 
+
+To change the scope of the RBAC permissions from cluster-wide to a specific namespace, you will need to use a `Role` instead of `ClusterRole` and `RoleBinding` instead of c
```

Comment:
@camilamacedo86 I think you missed this from eric's last review
```suggestion
To change the scope of the RBAC permissions from cluster-wide to a specific namespace, you will need to use `Role`s and `RoleBinding`s instead of `ClusterRole`s and `ClusterRoleBinding`s, respectively.
```